CMakeLists.txt

```diff
@@ -8,16 +8,36 @@ add_subdirectory(lib/googletest-master)
-        src/main.cpp
-        test/logger_test.cpp src/logger/logger.cpp src/logger/logger.h src/logger/loggerException.cpp src/logger/loggerException.h
-        test/config_test.cpp src/config/Config.cpp src/config/Config.h src/Game.h src/Game.cpp src/TextureManager.h src/TextureManager.cpp
-        src/gameobjects/Player.h src/gameobjects/Player.cpp src/gameobjects/GameObject.cpp src/gameobjects/GameObject.h
-        src/Factory/Factory.cpp src/Factory/Factory.h src/gameobjects/Coin.cpp src/gameobjects/Coin.h src/GameMap.cpp src/GameMap.h test/gamemap_test.cpp src/gameobjects/EnemyMushroom.cpp src/gameobjects/EnemyMushroom.h src/gameobjects/EnemyTurtle.cpp src/gameobjects/EnemyTurtle.h src/gameobjects/PlatformNormal.cpp src/gameobjects/PlatformNormal.h src/gameobjects/PlatformSurprise.cpp src/gameobjects/PlatformSurprise.h
-        src/Camera.cpp src/Camera.h src/BackgroundStages/FirstStage.cpp src/BackgroundStages/FirstStage.h src/BackgroundStages/BackgroundStage.h
-        src/BackgroundStages/BackgroundStage.cpp src/BackgroundStages/SecondStage.cpp src/BackgroundStages/SecondStage.h src/CharacterStates/CharacterState.cpp
-        src/CharacterStates/CharacterState.h src/CharacterStates/Normal.cpp src/CharacterStates/Normal.h src/CharacterStates/Jumping.cpp src/CharacterStates/Jumping.h
-        src/CharacterStates/Running.cpp src/CharacterStates/Running.h src/Utils/Vector.cpp src/Utils/Vector.h)
+        src/main.cpp src/main.h
+        src/logger/logger.cpp src/logger/logger.h
+        src/logger/loggerException.cpp src/logger/loggerException.h
+        src/config/Config.cpp src/config/Config.h
+        src/utils/Printer.cpp src/utils/Printer.h
+        src/Game.cpp src/Game.h
+        src/GameMap.cpp src/GameMap.h
+        src/TextureManager.cpp src/TextureManager.h
+        src/Camera.cpp src/Camera.h
+        src/BackgroundStages/BackgroundStage.cpp src/BackgroundStages/BackgroundStage.h
+        src/BackgroundStages/FirstStage.cpp src/BackgroundStages/FirstStage.h
+        src/BackgroundStages/SecondStage.cpp src/BackgroundStages/SecondStage.h
+        src/CharacterStates/CharacterState.cpp src/CharacterStates/CharacterState.h
+        src/CharacterStates/Jumping.cpp src/CharacterStates/Jumping.h
+        src/CharacterStates/Normal.cpp src/CharacterStates/Normal.h
+        src/CharacterStates/Running.cpp src/CharacterStates/Running.h
+        src/Factory/Factory.cpp src/Factory/Factory.h
+        src/gameobjects/Coin.cpp src/gameobjects/Coin.h
+        src/gameobjects/EnemyMushroom.cpp src/gameobjects/EnemyMushroom.h
+        src/gameobjects/EnemyTurtle.cpp src/gameobjects/EnemyTurtle.h
+        src/gameobjects/GameObject.cpp src/gameobjects/GameObject.h
+        src/gameobjects/PlatformNormal.cpp src/gameobjects/PlatformNormal.h
+        src/gameobjects/PlatformSurprise.cpp src/gameobjects/PlatformSurprise.h
+        src/gameobjects/Player.cpp src/gameobjects/Player.h
+        src/utils/Timer.cpp src/utils/Timer.h
```

Comment:
Timer is at Utils, maybe we should join the folder utils and Utils so we avoid this problem in a future?

Reply:
Habla en español gil

Reply:
Listo! Surgió al traerme los cambios de master ya había también otro utils con mayúscula.